src/v2/components/TourDeSol/index.jsx

```diff
@@ -5,40 +5,111 @@ import cn from 'classnames';
 import {observer} from 'mobx-react-lite';
 import SectionHeader from 'v2/components/UI/SectionHeader';
 import iconRight from 'v2/assets/icons/arrow-right-dark.png';
-
 import Ranking from './Ranking';
 import Table from './Table';
 import Cards from './Cards';
+import OverviewStore from 'v2/stores/networkOverview';
+
 import useStyles from './styles';
+import _ from 'lodash';
+import moment from 'moment';
+
+const SLOTS_PER_DAY = (1.0 * 24 * 60 * 60) / 0.8;
```

Comment:
`0.8` should arrive here by RPC somehow.   For example on 0.17 (and TdS stage 1) it's now 0.4.  Might change again in the future...

Reply:
Hmmm, using `process.env.TDS_ACTIVE_STAGE` in the front-end like this doesn't work in production as webpack embeds the value at build time (which is when the npm module is shipped from Travis CI).  

This is what runs in production:
https://github.com/solana-labs/blockexplorer/blob/4643594587599d3c3a4c1f2390a75e285fc039de/package.json#L111

Reply:
@mvines sounds good re: the slot constant - I will send that from the API when I add the RPC endpoint for inflation rate.

Should I send the TDS active stage from the API as well while I'm there? (Wondering if the API script can have that environment variable)

Reply:
Yeah an API server could pull out env vars.   So on the TdS API server we could define some TDS-specific vars, there's a bit of plumbing in the main repo we need for that but I can help do that easily enough.   But so this also implies that we need to make the display of the TdS front-end pages contingent on whether the API server it's currently talking to is "TdS enabled" -- which generally seems like the right thing to do anyway :)

Reply:
OK I'll merge this as part 1 and come back to the API stuff when I plumb in the inflation rate (early next week?)